src/detectors/BEMC/BEMC.cc

```diff
@@ -95,7 +95,11 @@ extern "C" {
           new JOmniFactoryGeneratorT<CalorimeterClusterRecoCoG_factory>(
              "EcalBarrelScFiClusters",
             {"EcalBarrelScFiProtoClusters",        // edm4eic::ProtoClusterCollection
+#if EDM4EIC_VERSION_MAJOR >= 7
+             "EcalBarrelScFiRawHitAssociations"},  // edm4eic::MCRecoCalorimeterHitAssociation
```

Comment:
Can we already put this in for the EcalBarrelImagingClusters, even though I suspect EcalBarrelImagingRawHitAssociations is missing?

Reply:
Hmmm... Good question 😕 

But that's correct: right this PR doesn't update the ImagingClusterReco algorithm, and there's no PR open (right now) to update it. So would it be better to keep this focused on ClusterRecoCoG and open a parallel PR for ImagingClusterReco, or make the change in this one?

Reply:
The imaging calorimeter also uses ClusterRecoCoG. The differences are only upstream, in how proto clusters are found. In that sense this is independent of which kind of clustering is used.

Reply:
Ah! Do the Imaging Clusters also use RecoCoG? I could've sworn they used ImagingClusterReco...

Reply:
Yeah, you're right.

Reply:
In that case, it looks like the logic is identical to ClusterRecoCoG. So it should be pretty straightforward to swap out what's in ImagingClusterReco with what I have in this PR!

Reply:
Marking resolved for this is out of scope.

Reply:
Sounds good. I'll put in a parallel PR for the ImagingClusterReco.